src/openrpc/localization.json

```diff
@@ -132,6 +132,11 @@
       "summary": "Get the ISO 639 1/2 code for the preferred language",
       "params": [],
       "tags": [
+        {
```

Comment:
also deprecate setLangauge and onLanguageChanged

Reply:
These are autogenerated, and will inherit the deprecation upon generation.